### DISCUSSION TITLE
Add bulk-edit for icons configuration

### DISCUSSION THREAD

**Discussion by abriemme** [home-assistant/frontend#10623] [Configuration]
It would save me (and others maybe) a lot time configuring entities icons in bulk.
For now i have to do it one by one.
We already have the capability to select multiple entities (to enable/disable in bulk).

I'm thinking something like: 
<img width="1185" alt="Screenshot 2021-11-15 at 12 33 50" src="https://user-images.githubusercontent.com/3003694/141775535-66dcf150-8dd3-41e4-b39b-e37948bd4ed0.png">

**Comment by Wendelstein7** [comment]
Hi! I am very, very appreciative of all the work that people have put in the updates with mass selection of entities. This feature request is quite old, but I would like to bump it given the new context and actual developments, as this feature request is almost 3 years old.

Currently, you have a few options when selecting multiple entities on the top right: "Add label" and a triple-dot dropdown to manage enabledness and visibility. **How cool would it be to also allow mass setting icons from here?** It would greatly help me out, as I can easily query for specific entities using filters ad the search bar, and then set 40 or 50 icons at once instead of opening the dialog 40 to 50 times.

![image](https://github.com/home-assistant/frontend/assets/23213464/81dd7a30-c4db-4a3c-8202-86b332216a67)

Let me know what you think! I tried to but didn't manage to succesfully setup the home assitant development on my local system so with my skill level unfortunately I am unable to help out myself. But perhaps someone who can would like this feature too! <3

**Reply by psychuk** [reply]
I just looked for this very thing.
It's a shame ideas don't get picked up so quickly here, not a single comment from a dev.
What harm would an 'Icon' dropdown do?

**Reply by Wendelstein7** [reply]
I don't think it is a matter of 'harm', but more one of work needed. Things like this need work and skill (otherwise I or you could've done it) and I suppose it is not a priority for one of the people who has the time and skill. Maybe eventually, but I'd love to hear what the Nabu Casa engineers thing!

**Reply by karwosts** [reply]
Just a couple stray thoughts:

- Most of the feature request activity has migrated to https://github.com/orgs/home-assistant/discussions, there's not as much visibility here anymore.
- In the year or so that's been open, I don't think anyone has requested this feature there.
- In the ~5 years or so this has been open, it's only got a handful of upvotes, so it seems like it's not terribly interesting to a lot of people.
- I'm a bit on the fence about the idea, it's probably not difficult to add, but I'm not sure how much it is a natural fit with the other bulk items, and I'm not really sure I can imagine a lot of usecases for it. Maybe it's just me but I've not too often felt like I had a desire to re-icon dozens of entities at once.



Anyway if this is something important to you, I would recommend to open a req in the new feature request repo, and include some examples about why this would be useful. Maybe can drive some more upvotes :shrug:
